### ISSUE TITLE
The app doesn't run on Linux arm64

### ISSUE THREAD

**Issue by real-felix** [Automattic/simplenote-electron#3207]
When I run the app on my system, the app crashes at once. Here is the error:

```
Simplenote-linux-2.21.0-arm64> ./simplenote 
fish: Job 1, './simplenote' terminated by signal SIGSEGV (Address boundary error)
```

### Where did you see the bug

I downloaded the arm64 archive.

- System Make: ?
- System Model: Linux 6.8.9-404.asahi.fc40.aarch64+16k
- OS: Fedora Linux Asahi Remix 40 (Forty)
- Simplenote app version: 2.21.0

**Comment by codebykat** [comment]
Hi @real-felix, thanks for the report. This should be fixed in the upcoming 2.22 release. You can try it from here: https://github.com/Automattic/simplenote-electron/releases/tag/v2.22.0-beta1

Let me know if this still happens on the new version.

**Comment by real-felix** [comment]
It works perfectly, thanks!

It's amazing how you do a better job that some multi-billions companies that cannot release an ARM desktop app.
